docs/reach/rs2/led-status.md

```diff
@@ -1,48 +1,60 @@
-|--------------|-------|
-| <div style="text-align: center;"> **Rover statuses** </div> |
-| <br><br><br><div style="width:400px;text-align: center;">    App not started <br>or no RTK solution available   </div> |    <br>  <div style="text-align: center;"><img src="../img/reachrs2/led-status/no-solution.png" style="height: 150px;"></div>    |
-| <br><br><br><br><div style="text-align: center;">    Single   </div> |    <br>  <div style="text-align: center;"><img src="../img/reachrs2/led-status/single.gif" style="height: 150px;"></div>    |
-| <br><br><br><br><div style="text-align: center;">    Float   </div> |    <br>  <div style="text-align: center;"><img src="../img/reachrs2/led-status/float.gif" style="height: 150px;"></div>    |
-| <br><br><br><br><div style="text-align: center;">    Fix   </div> |    <br>  <div style="text-align: center;"><img src="../img/reachrs2/led-status/fix.png" style="height: 150px;"></div>   |
-| <br><br><br><br><div style="width:400px;text-align: center;">    Point collection </div> |    <br>  <div style="text-align: center;"><img src="../img/reachrs2/led-status/point-collection.gif" style="height: 150px;"></div>    |
-| <div style="text-align: center;"> **Base statuses** </div> |
-| <br><br><br><br><div style="text-align: center;">    Correction output ON   </div> |    <br>  <div style="text-align: center;"><img src="../img/reachrs2/led-status/base-mode.png" style="height: 150px;"></div>   |
-| <br><br><br><br><div style="text-align: center;">    Both Correction Input and Correction output ON, <br>  the device holds fix   </div> |    <br>  <div style="text-align: center;"><img src="../img/reachrs2/led-status/correction-in-out.gif" style="height: 150px;"></div>   |
+![](../img/reachrs2/led-status/rs2.png){.reciever}
+
+<div class="reciever-wrapper">    
+<div class="reciever-container">
+<div class="reciever-description">
+    <h4>Battery</h4>
+    <p>The lights above the power button indicate the battery charge level. If the loading animation is over, then you can connect your phone to Reach.</p>
+</div>
+<div class="reciever-status">
+    <div class="status-wrapper"><img src="../img/reachrs2/led-status/booting.gif"><span>Loading</span></div>
+    <div class="status-wrapper"><img src="../img/reachrs2/led-status/full-charge.jpg"><span>Full charge</span></div>
+    <div class="status-wrapper"><img src="../img/reachrs2/led-status/charging.gif"><span>Charging</span></div>
+    <div class="status-wrapper"><img src="../img/reachrs2/led-status/low-battery.jpg"><span>Low battery</span></div>
+    <div class="status-wrapper"><img src="../img/reachrs2/led-status/Nech_boot.gif"><span>Not enough charge to boot</span></div>
+</div>
+</div>
+<div class="reciever-container">
+    <div class="reciever-description"> 
+    <h4>Network</h4>
+    <p>During boot, Reach RS2 enters a network scan state and searches for known networks to connect to. If it didn’t find a known network, it switches to hotspot mode.</p>
```

Comment:
_If it didn’t find a known network, it switches to hotspot mode._ => If it doesn't find a known network, it switches to hotspot mode.